migration/v1_to_v2/configuration/exporters/form_config_exporter.rb

```diff
@@ -29,31 +30,41 @@ def default_service_referral_fields
         'type' => 'tick_box',
         'tick_box_label_en' => 'Yes',
         'visible' => false,
-        'display_name_en' => 'Is this a referral to an external system / user?'
+        'display_name_i18n' => {
```

Comment:
why do you need all these changes, v2 code will insert the values in the right structure